src/commands/docker_build_push.yml

```diff
@@ -0,0 +1,15 @@
+description: >
+  Build the docker image with docker.
+
+parameters:
+  work_dir:
+    type: string
+    default: .
+
+steps:
+  - run:
+      name: 'docker build'
+      command: docker build -t ric-docker.jfrog.io/ricardo-ch/search-solr:${CIRCLE_BRANCH}-${CIRCLE_SHA1} .
+  - run:
+      name: 'docker push'
+      command: docker push ric-docker.jfrog.io/ricardo-ch/search-solr:${CIRCLE_BRANCH}-${CIRCLE_SHA1}
```

Comment:
In theory, we could also pass the Dockerfile as a param (iirc `docker` has  a `-f`) but perhaps it's too much?

Reply:
lets do it when we need it